packages/server/lib/gui/events.ts

```diff
@@ -23,141 +23,13 @@ const handleEvent = function (options, bus, event, id, type, arg) {
 
   switch (type) {
```

Comment:
@Ashviniv Thank you for this contribution! Looks like removing this leaves this file as a shell/placeholder and it is not actually being leveraged anymore. Can we do the full clean up to remove this file completely?